concord-ui/src/components/notfound.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import '@rmwc/data-table/data-table.css';
+
+export class NotFound extends Component {
+  render() {
+    return (
+      <div>
+        <p>404 Not Found</p>
```

Comment:
Surely a bit of styling? Also would be nice to add a 'return to home' button here.